CodeSniffer/Standards/Squiz/Sniffs/WhiteSpace/MemberVarSpacingSniff.php

```diff
@@ -34,6 +34,16 @@ class Squiz_Sniffs_WhiteSpace_MemberVarSpacingSniff extends PHP_CodeSniffer_Stan
 
 
     /**
+     * The exception to the newline requirement on the first member var after class opener.
+     * false for newline requirement on the first member var after class opener
+     * true for no newline requirement on the first member var after class opener
+     *
+     * @var bool
+     */
+    public $firstVarException = false;
```

Comment:
Maybe I'm not seeing what you're seeing, How is this a BC break?

What I see is a new API end point adding additional functionality which is controlled by a new member var parameter. 

The existing behavior for this sniff is unchanged... unless you set the new member var parameter to true thus turning on the exception.